packages/@aws-cdk/aws-eks/README.md

```diff
@@ -386,6 +386,20 @@ A convenience method for mapping a role to the `system:masters` group is also av
 cluster.awsAuth.addMastersRole(role)
 ```
 
+### Cluster Security Group
```

Comment:
What about the encryption config?

Reply:
i didn't add that before because you actually pass the config as input during cluster creation. but just for the sake of completion, now is there